jupyter/datascience/ubi9-python-3.11/Pipfile

```diff
@@ -7,44 +7,50 @@ name = "pypi"
 
 [packages]
 # Datascience and useful extensions
-boto3 = "~=1.34.50"
+boto3 = "~=1.35.5"
 kafka-python = "~=2.0.2"
-matplotlib = "~=3.8.3"
-numpy = "~=1.26.4"
-pandas = "~=2.2.0"
-plotly = "~=5.20.0"
-scikit-learn = "~=1.4.0"
-scipy = "~=1.12.0"
-skl2onnx = "~=1.16.0"
-onnxconverter-common = "~=1.13.0" # Required for skl2onnx, as upgraded version is not compatible with protobuf
+matplotlib = "~=3.9.2"
+numpy = "~=2.1.0"
+pandas = "~=2.2.2"
+plotly = "~=5.23.0"
+scikit-learn = "~=1.5.1"
+scipy = "~=1.14.1"
+skl2onnx = "~=1.17.0"
+onnxconverter-common = "~=1.14.0" # Required for skl2onnx, as upgraded version is not compatible with protobuf
 codeflare-sdk = "~=0.18.0"
 memray = "~=1.10.0"  # any version above this would cause issue with rich package
+
 # DB connectors
-pymongo = "~=4.6.2"
-psycopg = "~=3.1.18"
+pymongo = "~=4.8.0"
+psycopg = "~=3.2.1"
 pyodbc = "~=5.1.0"
-mysql-connector-python = "~=8.3.0"
+mysql-connector-python = "~=9.0.0"
+
 # JupyterLab packages
-odh-elyra = "~=3.16.7"
-jupyterlab = "~=3.6.7" # Wait on upgrade till plugins are ready
-jupyter-bokeh = "~=3.0.7" # Upgrade would bring in jupyterlab 4
-jupyter-server = "~=2.13.0"
-jupyter-server-proxy = "~=4.0.0" # Upgrade would bring in jupyterlab 4
+
+#odh-elyra = ">=3.16.7" # For the phase 1, exclude elyra installation. Till this Jira ticket is resolved. https://issues.redhat.com/browse/RHOAIENG-10143
+
+jupyterlab = "~=4.2.4"
+jupyter-bokeh = "~=4.0.5"
+jupyter-server = "~=2.14.2"
+jupyter-server-proxy = "~=4.3.0"
 jupyter-server-terminals = "~=0.5.3"
-jupyterlab-git = "~=0.44.0"
-jupyterlab-lsp = "~=4.2.0"
-jupyterlab-widgets = "~=3.0.10"
-jupyter-resource-usage = "~=0.7.2"
-nbdime = "~=3.2.1"
-nbgitpuller = "~=1.2.0"
+jupyterlab-git = "~=0.50.1"
+jupyterlab-lsp = "~=5.1.0"
+jupyterlab-widgets = "~=3.0.13"
+jupyter-resource-usage = "~=1.1.0"
+nbdime = "~=4.0.1"
+nbgitpuller = "~=1.2.1"
+
 # pycodestyle is dependency of below packages
 # and to achieve compatible of pycodestyle with python-lsp-server[all]
 # pinned the below packages
-autopep8 = "~=2.0.4"
-flake8 = "~=7.0.0"
+autopep8 = "~=2.3.1"
+flake8 = "~=7.1.1"
+
```

Comment:
I believe these can be removed from direct list , as odh-elyra is removed.
they came from there.